src/plugins/aragon/index.ts

```diff
@@ -1,5 +1,113 @@
-import { sendTransaction } from '../../utils';
-import abi from './DisputableDelay.json';
+import BN from 'bn.js';
+import { keccak256 } from '@ethersproject/keccak256';
+import { toUtf8Bytes } from '@ethersproject/strings';
+import { sendTransaction, subgraphRequest } from '../../utils';
+import abi from './GovernQueue.json';
+
+const ARAGON_SUBGRAPH_URL = {
+  4: 'https://api.thegraph.com/subgraphs/name/evalir/aragon-govern-rinkeby'
+};
+
+const GQL_QUERY = {
+  governQueue: {
+    __args: {
+      id: '0xfc67dd2d7a67bc7b1aa5061c3f4b49f6c3341d00'
+    },
+    config: {
+      executionDelay: true,
+      scheduleDeposit: {
+        token: {
+          id: true
+        },
+        amount: true
+      },
+      challengeDeposit: {
+        token: {
+          id: true
+        },
+        amount: true
+      },
+      vetoDeposit: {
+        token: {
+          id: true
+        },
+        amount: true
+      },
+      resolver: true,
+      rules: true
+    }
+  }
+};
+
+const FAILURE_MAP =
+  '0x0000000000000000000000000000000000000000000000000000000000000000';
+const EMPTY_BYTES = '0x00';
+
+/**
+ * scheduleAction schedules an action into a GovernQueue.
+ * Instead of sending the action to a disputable delay from aragonOS, we directly call this
+ * contract.
+ * the actionsFromAragonPlugin is an array of objects with the form { to, value, data }
+ */
+async function scheduleAction(
+  web3,
+  governQueueAddress,
+  account,
+  proof,
+  executor,
+  actionsFromAragonPlugin
+) {
+  const config = await subgraphRequest(ARAGON_SUBGRAPH_URL[4], GQL_QUERY);
+
+  // Building the nonce for the next tx
+  const nonce = await web3.nonce();
+  const bnNonce = new BN(nonce.toString());
+  const newNonce = bnNonce.add(new BN('1'));
+  // We also need to get a timestamp bigger or equal to the current block.timestamp + config.executionDelay
+  // Right now + execution delay + 60 seconds into the future
+  const currentDate =
+    Math.round(Date.now() / 1000) + Number(config.executionDelay) + 60;
+
+  return await sendTransaction(
+    web3,
+    abi,
+    governQueueAddress,
+    'delayExecution',
```

Comment:
The method name on the contract is actually [`schedule`](https://github.com/aragon/govern/blob/master/packages/govern-core/contracts/pipelines/GovernQueue.sol).